src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx

```diff
@@ -100,10 +109,12 @@ function CommunityCareAppointmentDetailsPage({
       />
 
       <div className="vads-u-margin-top--3 vaos-appts__block-label">
-        <AppointmentInstructions
-          instructions={instructions}
-          isHomepageRefresh
-        />
+        <div className="vads-u-flex--1 vads-u-margin-bottom--2 vaos-u-word-break--break-word">
```

Comment:
Based on the screenshot, I think we have some extra info in the header that shouldn't be there. This `instructions` object structure was made up to make some things easier across different appointment types, but I think it's causing confusion in here, where the logic is much simpler than for VA appointments. I think you could do:

```
{!!appointment.comment && (<div className="vads-u-flex--1 vads-u-margin-bottom--2 vaos-u-word-break--break-word">
    <h2 className="vads-u-font-size--base vads-u-font-family--sans vads-u-margin-bottom--0">
        Special instructions
    </h2>
    <div>{appointment.comment}</div>
</div>)}
```

Reply:
Done